src/test/java/ru/ayaz/CommandExecutorParseTest.java

```diff
@@ -28,7 +29,6 @@ public void shouldGetCommandWhenStringHasTwoWords() {
         assertEquals("#ignore", getCommand("#ignore spammer"));
     }
 
-
     @Test
     public void shouldGetArgumentWhenStringHasOneArgument() {
```

Comment:
&#x1F34E; **Issue**
You shouldn't reveal such an implementation details just for the sake of testing.
You should test this functionality through your public interface only.
You should be testing not particular class methods, but rather the use-cases your class is intended for from the Domain logic perspective.